CMakeLists.txt

```diff
@@ -454,9 +456,9 @@ if(HPX_WITH_IO_COUNTERS)
       "HPX_WITH_IO_COUNTERS was set to ON, but IO counters are only available on Linux (this is \"${CMAKE_SYSTEM_NAME}\")"
     )
   endif()
-  if(NOT HPX_WITH_DISTRIBUTED_RUNTIME)
+  if(NOT HPX_HAVE_DISTRIBUTED_RUNTIME)
     hpx_error(
-      "HPX_WITH_IO_COUNTERS was set to ON, but performance counters cannot be used with HPX_WITH_DISTRIBUTED_RUNTIME=OFF"
+        "HPX_WITH_IO_COUNTERS was set to ON, but performance counters cannot be used with HPX_WITH_NETWORKING=OFF or HPX_HAVE_DISTRIBUTED_RUNTIME=OFF"
```

Comment:
This is not true. They can be used with `HPX_WITH_NETWORKING=OFF`. Same below for PAPI.